### PR TITLE
[FR] Add dataPriority option to columns

### PR DESCRIPTION
**Pull request by adlerweb**
When using the [responsive datatables extension](https://datatables.net/extensions/responsive/) columns automatically hide on small screens. Usually the rightmost columns are hidden first, however it is possible to specify a priority for each column to change this behavior. At the moment such a priority can be set in datatables-bundle using a [configuration option](https://datatables.net/extensions/responsive/priority#Configuration-option), however this uses column numbers, requires manual updates when adding or removing columns and is somewhat confusing to maintain. This PR adds a possibility to set the corresponding [column attribute](https://datatables.net/extensions/responsive/priority#Priority-data-attribute) using a column configuration.

### CODE REVIEW

File: docs/source/index.html.md [hunk 1]
```diff
@@ -402,6 +402,7 @@ searchable | bool/`null` | Whether the column can be searched upon. Defaults to
 globalSearchable | bool/`null` | Whether the column participates in global searches. Defaults to the presence of `field`.
 className | string/`null` | A CSS class to be applied to all cells in this column.
 render | string/callable/`null` | Either a [`sprintf` compatible format string](http://php.net/manual/en/function.sprintf.php), or a callable function providing rendering conversion, or default `null`.
+dataPriority | integer/`null` | A priorty for this column. When using a responsive table higher numbers disappear first.
```
Comment (curry684) [inline docs/source/index.html.md]: Typo in priority

**Comment by adlerweb** [comment]
> Extension makes sense, not really sure I like the `dataPriority` name. Shouldn't we just have a free-form `data` array for this purpose?

Hm, only for data-* or should it be a generic "attributes"-array, risking collisions with arguments automatically generated?

**Comment by curry684** [comment]
`data` only, we don't want to interfere with 'native' behavior.

**Comment by curry684** [comment]
@shades684 agree with this?

**Comment by curry684** [comment]
Agreeing with @shades684 and closing.

**Pull request closed without merging**